src/pages/home/report/ReportFooter.js

```diff
@@ -57,53 +57,47 @@ const defaultProps = {
     shouldDisableCompose: false,
 };
 
-class ReportFooter extends React.Component {
-    /**
-     * @returns {Object}
-     */
-    getChatFooterStyles() {
-        return {...styles.chatFooter, minHeight: !this.props.isOffline ? CONST.CHAT_FOOTER_MIN_HEIGHT : 0};
-    }
+function ReportFooter(props) {
+    const getChatFooterStyles = () => ({...styles.chatFooter, minHeight: !props.isOffline ? CONST.CHAT_FOOTER_MIN_HEIGHT : 0});
```

Comment:
Please rewrite this as a const style, no need for a function here.

Reply:
It's now a function and passed to `View` as a function, it's not worked as expected.